onboarding/src/Components/EmployeeProfile/ImplementationFormsSent.js

```diff
@@ -124,12 +41,11 @@ function ImplementationFormsSent(props) {
     return(
         <div className="card card-fluid">
             <div className="card-header">
-                <i className="bi bi-cloud-check mr-2" style={{ fontSize: "18px" }}></i> Wysłane katalogi wdrożeniowe
+                <i className="bi bi-cloud-check mr-2" style={{fontSize: "18px"}}/> Wysłane katalogi wdrożeniowe
             </div>
             <div className="card-body">
-                { error && <p>Wystąpił błąd podczas ładowania</p> }
-                { loading && <p>Ładowanie...</p> }
```

Comment:
Why loading is deleted? There should be info for user that data is not yet loaded. 
For now it firstly shows that there is no sent catalogues and then eventually shows them. It's a bit confusing, especially if internet connection is weak.

Reply:
Good idea :slightly_smiling_face: 
let me fix this. I think I know how to do it and I can do it fast ;)